keras/applications/inception_v4.py

```diff
@@ -0,0 +1,310 @@
+from __future__ import print_function
+from __future__ import absolute_import
+
+import warnings
+
+from ..models import Model
+from .. import layers
+from ..layers import Activation
+from ..layers import Dense
+from ..layers import Input
+from ..layers import Conv2D
+from ..layers import MaxPooling2D
+from ..layers import AveragePooling2D
+from ..layers import BatchNormalization
+from ..layers import Dropout
+from ..layers import Flatten
+from .. import regularizers
+from .. import initializers
+from .. import backend as K
+from ..utils import convert_all_kernels_in_model
+from ..utils import get_file
+
+#########################################################################################
+# Implements the Inception Network v4 (http://arxiv.org/pdf/1602.07261v1.pdf) in Keras. #
+#########################################################################################
+
+WEIGHTS_PATH = 'https://github.com/kentsommer/keras-inceptionV4/releases/download/2.1/inception-v4_weights_tf_dim_ordering_tf_kernels.h5'
```

Comment:
Should this be hosted on `fchollet/deep-learning-models` like the other models?

Reply:
@bdwyer2 Probably

Reply:
Maybe you should open a PR on that repository too